suites/apis/guppyTest.js

```diff
@@ -0,0 +1,87 @@
+Feature('Guppy')
+
+const chai = require('chai');
+const expect = chai.expect;
+const apiUtil = require('../../utils/apiUtil.js');
+const guppy = require('../../services/apis/guppy/guppyService.js');
+const sheepdog = require('../../services/apis/sheepdog/sheepdogService.js');
+const nodes = require('../../utils/nodes.js');
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+Before( async (home, etl, nodes) => {
+  home.complete.login();
+});
+
+// This test fails due to what looks to me like a bug in Guppy that should be fixed. I made a ticket for it.
+xScenario('I want to make a query to Guppy, but my access token is invalid or expired.', async (I, guppy) => {
+  let invalidToken = 'eyJhbGciOiJSUzI1N';
+  const queryFile = 'test_plans/guppy/test_data/test_query_1.json';
+  const queryResponse = await guppy.do.submitQueryFileToGuppy(guppy.props.endpoints.graphqlEndpoint, queryFile, invalidToken);
+  expect(queryResponse.status).to.equal(401);
+});
+
+Scenario('I want a list of patients (cases) strictly younger than 30 with a past stroke in ascending order of BMI.', async (I, guppy) => {
+  let token = await I.grabCookie('access_token');
```

Comment:
you could do something similar to this https://github.com/uc-cdis/gen3-qa/blob/master/suites/apis/userTokenTest.js#L24 to get the access token

Reply:
done